rvzr/model_dynamorio/model.py

```diff
@@ -87,16 +89,18 @@ def load_test_case(self, test_case: TestCaseProgram) -> None:
         # store the test case in the RCBF format
         test_case.get_obj().save_rcbf(self._rcbf_file)
 
-    def trace_test_case(self, inputs: List[InputData], nesting: int) -> List[CTrace]:
+    def trace_test_case_with_addr(self, inputs: List[InputData],
```

Comment:
This is not a new API for the model, correct? Then rename to "_trace_test..." to label it as a private method